### ISSUE TITLE
Flake in TestPolicyBasedRestrictionOfBuildConfigCreateAndInstantiateByStrategy

### ISSUE THREAD

**Issue by smarterclayton** [openshift/origin#5449]
```
I1026 18:53:23.943824     452 helper.go:366] UpdateTrackingTags: stream=integration/builderimage, updatedTag=latest, updatedImage.dockerImageReference=test/builderimage:latest, updatedImage.image=image-id
--- FAIL: TestPolicyBasedRestrictionOfBuildConfigCreateAndInstantiateByStrategy-2 (6.44s)
	build_admission_test.go:171: Couldn't create ImageStreamMapping: imageStream "builderimage" cannot be updated: the object has been modified; please apply your changes to the latest version and try again
FAIL
```

**Comment by smarterclayton** [comment]
https://ci.openshift.redhat.com/jenkins/job/test_pull_requests_origin/6274/

**Comment by deads2k** [comment]
@smarterclayton Doesn't this look the same as all the other flakes failing on the ISM update that @ironcladlou is looking at?

**Comment by deads2k** [comment]
Looks like a dupe of https://github.com/openshift/origin/issues/5421

**Comment by deads2k** [comment]
resolved with #5455
